### PR TITLE
remove sbt-cross-building, add .travis.yml

### PR DESCRIPTION
**Pull request by xuwei-k**
fix https://github.com/jrudolph/sbt-dependency-graph/issues/121

@jrudolph could you enable travis-ci?

### CODE REVIEW

**Comment by xuwei-k** [comment]
succeeded in my fork https://travis-ci.org/xuwei-k/sbt-dependency-graph/builds/239606357

File: src/sbt-test/sbt-dependency-graph/intervalRangedVersions/build.sbt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 scalaVersion := "2.9.1"
 
+resolvers += "typesafe maven" at "http://repo.typesafe.com/typesafe/maven-releases/"
```
Comment (jrudolph) [inline src/sbt-test/sbt-dependency-graph/intervalRangedVersions/build.sbt]: Do we really need the extra resolver?

**Comment by jrudolph** [comment]
Thanks a lot @xuwei-k.

File: src/sbt-test/sbt-dependency-graph/intervalRangedVersions/build.sbt (hunk 1 above)
Comment (xuwei-k) [inline src/sbt-test/sbt-dependency-graph/intervalRangedVersions/build.sbt]: without resolver

```
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 	::          UNRESOLVED DEPENDENCIES         ::
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [warn] 	:: com.codahale#jerkson_2.9.1;0.5.0: not found
[info] [warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[info] [info] Done updating.
```

- http://repo.typesafe.com/typesafe/maven-releases/com/codahale/jerkson_2.9.1/
- http://repo1.maven.org/maven2/com/codahale/jerkson_2.9.1/

Comment (jrudolph) [inline src/sbt-test/sbt-dependency-graph/intervalRangedVersions/build.sbt]: I guess it's also time to update those tests...

Comment (benmccann) [inline src/sbt-test/sbt-dependency-graph/intervalRangedVersions/build.sbt]: You should always use https for Maven repos: http://www.ontoillogical.com/blog/2014/07/28/how-to-take-over-any-java-developer/

**Pull request merged**